docs/installation.rst

```diff
@@ -1,50 +1,56 @@
 Installation
 ============
 
-You can run the webserver locally.  For now, static data is loaded for testing
-and development.
+You can work on the UI without needing a VM, by using web-server.js.
+There are a few limitations, since URL rewriting is not supported (and so some links will be broken), but it works well enough for quick testing. For instructions on how to serve the UI with working URL rewriting, use the treeherder-service Vagrant instructions.
 
+Cloning the Repo
+----------------
+
+* Clone the `treeherder-ui repo`_ from Github.
 
 Requirements
 ------------
 
-* node.js: http://nodejs.org/download/
+* Node.js_
 
-Execution::
+Running the web-server
+----------------------
 
-    cd webapp
-    ./scripts/web-server.js
+* Open a shell, cd into the root of the repository you just cloned and type:
 
+  .. code-block:: bash
 
-Endpoints
----------
+     cd webapp
+     cp app/js/config/sample.local.conf.js app/js/config/local.conf.js
+     ./scripts/web-server.js
 
-Once the server is running, you can nav to:
+Viewing the UI
+--------------
 
-* Jobs list: http://localhost:8000/app/index.html?tree=Try#/jobs
-* Log Viewer: http://localhost:8000/app/logviewer.html
+Once the server is running, you can nav to:
 
+* Jobs list: `<http://localhost:8000/app/index.html#/jobs?repo=try>`_
+* Log Viewer: `<http://localhost:8000/app/logviewer.html>`_
```

Comment:
When navigating to this guy without a revision, the Log Viewer looks a bit odd and might trigger new-user questions. Perhaps the first jobs link is enough?

![logviewerlink](https://cloud.githubusercontent.com/assets/3660661/6251586/c19481b4-b762-11e4-8163-31bd86a51020.jpg)